lib/json.rs

```diff
@@ -80,18 +94,6 @@ impl<'de> Deserialize<'de> for ValueWrapper {
     }
 }
 
-impl From<ValueWrapper> for JsonValue {
```

Comment:
Is this where the benefit is coming from?

Reply:
yes